tox.ini

```diff
@@ -23,6 +23,8 @@ deps =
     dj18: Django>=1.8,<1.8.99
     dj111: Django>=1.11a1,<1.11.99
     dj20: Django>=2.0.0,<2.0.99
+    pil27: pillow==2.7.0
```

Comment:
Very last comment (everything else looks great): this line and the one below it (`pil54: ...`) can be removed now that they aren't in the env specifiers.

Reply:
Updated 👍